marvin.py

```diff
@@ -140,27 +181,41 @@ def start(self, bot, update):
         @:param bot: an object that represents a Telegram Bot.
         @:param update: an object that represents an incoming update.
         """
-        update.message.reply_text('In un gruppo, rispondi ad un link con il comando /postlink')
+        if update.message.chat.id != self.authorized_group_id:
+            update.message.reply_text('Ciao, benvenuto in marvin! Visita la pagina '
+                                      'github per maggiori informazioni https://github.com/fen0x/marvin')
+        else:
+            self.delete_message_if_admin(update.message.chat, update.message.message_id)
+
+        return
 
     def comment(self, bot, update):
         """ (Telegram command)
         Adds a comment to a reddit post (only if it belong to the authorized subreddit)
         :param bot: an object that represents a Telegram Bot.
         :param update: an object that represents an incoming update.
         """
-        # Check if the command is used as reply to another message
-        if not update.message.reply_to_message:
-            update.message.reply_text("Per usare questo comando devi rispondere ad un messaggio")
-            return
+
         # Check if the command has been used in the correct group
         if not self.is_message_in_correct_group(update.message.chat):
-            update.message.reply_text("Spiacente, questo bot funziona solo nel gruppo autorizzato")
+            self.delete_message_if_admin(update.message.chat, update.message.message_id)
+            self.updater.bot.send_message(update.message.from_user.id,
+                                          "Spiacente, questo bot funziona solo nel gruppo autorizzato, non in " +
+                                          str(update.message.chat.id))
```

Comment:
Ha senso stampare l'id della chat come "non in questo gruppo"?
Non viene fuori qualcosa come "non in 2949201"?

Reply:
E' un'informazione utile, anche in caso che qualcuno usi il nostro bot, perché gli torna l'id che poi imposterà nella config

Reply:
Magari tra parentesi però mettiamo l'id della chat autorizzata, tanto lo hai nel JSON no? 
Dici che è fattibile?
Potremmo fare un messaggio di errore più completo come:
"nel gruppo autorizzato con id:<id> (chiamato <nome nel json>), non in <id>"

Reply:
va bene 👍 ora faccio

Reply:
Se lo fai come scritto qui poi copialo anche sotto, lo usi almeno 2 volte questo messaggio di errore 👍  grazie

Reply:
fatto

Reply:
nel mio caso esce

Spiacente, questo bot funziona solo nel gruppo autorizzato con id -1001401491585, non in 5651789

Reply:
Direi che può andar bene